### ISSUE TITLE
What's the status of this (VERY needed) project?

### ISSUE THREAD

**Issue by vanfanel** [tarvi-verro/weston-binder#2]
Hi there, Tarvi

I found out about this project some days ago. It's the final piece I need towards a fully functional Weston-based desktop!

So, what's going on with it? Is it abandoned in favor of other solution you may have found?

Thanks!

**Comment by tarvi-verro** [comment]
Hi vanfanel!

This was just a project I quickly created to see if Weston would work as a daily driver. I don't exactly recall why I chose to return to xfce4, but in the process this project became orphaned.

In any case, if people are still getting some use out of this codebase, I'd be happy to help with it

**Comment by vanfanel** [comment]
@tarvi-verro

With this project, Weston has became a daily driver for me!
If you are not maintaining it, I suppose it's not hard to define keybings manually on the Weston code?

Also, XFCE4 is what you use with Wayland nowadays? Does it work well without any X11 deps?

**Comment by tarvi-verro** [comment]
Yep - mocking up a Weston module just to add some keybindings was extremely simple. This was one of the reasons I actually wanted to stick to Weston - the codebase was really easy to hack.

An example of defining keybindings "manually" can actually be found in the first commit of this repo - https://github.com/tarvi-verro/weston-binder/commit/40b3112f221ef3a1c42f536d8047b8565ee3f2d8 

To answer the other question - I retreated back to X11 :(

**Comment by vanfanel** [comment]
> To answer the other question - I retreated back to X11 :(

Out of curiosity, why? X11 is HUGE and is a dead end after all.
A basic streamlined Wayland/Weston based desktop seems much better nowadays.

**Comment by keithbowes** [comment]
> Out of curiosity, why? X11 is HUGE and is a dead end after all. A basic streamlined Wayland/Weston based desktop seems much better nowadays.

Indeed.  I tried to get X back for testing purposes a while back and it was such a PITA to even get it to work (looking at the log file and trying to resolve all the errors).  Then when I did actually get it not to produce errors, I immediately noticed how much slower it was than Wayland.  Not to mention how much nicer Wayland libraries are compared to their X equivalents.

Though, Weston isn't my first choice.  If it had a taskbar, supported wlr-layer-shell, an had two clipboard buffers instead of one, it would be usable, but I'm guessing that those things will never happen, as Weston isn't really intended to be used but only as a demonstration.

**Comment by defdefred** [comment]
Hello @keithbowes, I tried to use your fork for libweston-10 but I can't start Weston with the modules, was it working for you?

Edit: it works now!
